tests/integration/rdb.tcl

```diff
@@ -4,6 +4,15 @@ set server_path [tmpdir "server.rdb-encoding-test"]
 
 # Copy RDB with different encodings in server path
 exec cp tests/assets/encodings.rdb $server_path
+exec cp tests/assets/old_ver.rdb $server_path
+
+start_server [list overrides [list "dir" $server_path "dbfilename" "old_ver.rdb"]] {
```

Comment:
```suggestion
start_server [list overrides [list "dir" $server_path "dbfilename" "list-quicklist-2.rdb"]] {
```
`old_ver.rdb` can't express it is quicklsit, we can change filename to `list-quicklist-2.rdb`.

Reply:
I assume you want to validate the old `RDB_TYPE_LIST_QUICKLIST` to the new `RDB_TYPE_LIST_QUICKLIST2`?
You can refer to the `convert-zipmap-hash-on-load` test.

Reply:
is my test not checking it correctly ?

Reply:
My mistake, the rdb name should be list-quicklist.rdb (the old rdb type).
I was actually wondering how to validate the old list-quicklist to list-quicklist-2 validation, since the encoding of the quicklist has not changed, do you have another way? All I can think of right now is to go through the quicklistRepr to get the relevant information.